node-src/lib/log.ts

```diff
@@ -4,47 +4,69 @@ import { createWriteStream, rm } from 'fs';
 import stripAnsi from 'strip-ansi';
 import { format } from 'util';
 
+import { Flags, Options } from '../types';
 import { errorSerializer } from './logSerializers';
 
 interface QueueMessage {
   type: LogType;
   messages: string[];
 }
 
-const { DISABLE_LOGGING, LOG_LEVEL = '' } = process.env;
 const LOG_LEVELS = { silent: 0, error: 1, warn: 2, info: 3, debug: 4 };
 const DEFAULT_LEVEL = 'info';
 
 // Top-level promise rejection handler to deal with initialization errors
 const handleRejection = (reason: string) => console.error('Unhandled promise rejection:', reason);
 process.on('unhandledRejection', handleRejection);
 
+const isPrintable = (value: any, type = typeof value) =>
```

Comment:
Do we need the `type` argument here? Should we just grab the type in the function itself?